servers/visual/visual_server_raster.h

```diff
@@ -149,7 +153,6 @@ class VisualServerRaster : public VisualServer {
 #define BINDBASE VSG::storage
 
 	/* TEXTURE API */
-
```

Comment:
It would be better to keep this separation, like done for other section comments.